src/rules/importBlacklistRule.ts

```diff
@@ -15,9 +15,7 @@
  * limitations under the License.
  */
 
-import {
-    isCallExpression, isExternalModuleReference, isIdentifier, isImportDeclaration, isImportEqualsDeclaration, isTextualLiteral,
-} from "tsutils";
+import { findImports, ImportOptions } from "tsutils";
```

Comment:
`ImportOptions` is a little opaque to me; I think a better name might be `ImportKind`?